ingest/expression_files/mtx.py

```diff
@@ -157,10 +157,24 @@ def check_bundle(barcodes, genes, mtx_dimensions):
 
         if (actual_barcodes == expected_barcodes) and (actual_genes == expected_genes):
             return True
+        elif (actual_barcodes == expected_genes) and (
+            actual_genes == expected_barcodes
+        ):
+            msg = (
+                f"Uploaded matrix suggests {actual_barcodes} cells and {actual_genes} genes "
+                f"instead of {expected_barcodes} cells and {expected_genes} genes. "
+                f"Please transpose your sparse matrix and re-upload. "
+                f"Transposition is expected for matrices exported from AnnData objects. "
```

Comment:
I think a more direct and active voice would decrease cognitive load, e.g.:

```suggestion
                f"Uploaded matrix has {actual_barcodes} cells and {actual_genes} genes "
                f"instead of {expected_barcodes} cells and {expected_genes} genes. "
                f"Please transpose your sparse matrix and re-upload. "
                f"Matrices exported from AnnData objects must be transposed."
```

Also, a tiny copyedit: there is no trailing period in the last sentence of the original message, so I imagine we should also omit it in this one for consistency.

Reply:
Great suggestion. Made a more technically correct modification - the uploaded matrix suggests column and row counts that don't match the counts for the cell and gene files so I'm leaving the word "suggests" and changing "cells" -> "columns" and "genes" -> "rows"
`6ba62e0`
Also, updated to be more faithful to generalized nomenclature "genes" -> "features/genes".
`71b083a`